### PR TITLE
Minor changes in Readme

### PR DESCRIPTION
**Pull request by megha070**
Signed-off-by: megha070 <varshney.megha070@gmail.com>

### CODE REVIEW

**Comment by megha070** [comment]
@bmpotter existing readme consists of postgres installation instructions for Mac OS but not Linux. Shall I include those too?

**Comment by megha070** [comment]
@bmpotter existing readme consists of postgres installation instructions for Mac OS but not Linux. Shall I include those too?

File: README.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ export EXCHANGE_ROOTPW=myrootpw
 ```
 export EXCHANGE_KEY_PW=<pass-phrase>
 make gen-key
+Note : pass-phrase can be any alphanumeric string
```
Comment (bmpotter) [inline README.md:56]: Are you sure special characters aren't allowed?

File: README.md [hunk 2]
```diff
@@ -65,9 +66,12 @@ make gen-key
 - To try a simple rest method curl: `curl -X GET "http://localhost:8080/v1/admin/version"`. You should get the exchange version number as the response.  
 - When testing the exchange in an OpenShift Cluster the variables `EXCHANGE_IAM_ORG`, `EXCHANGE_IAM_KEY` and `EXCHANGE_MULT_ACCOUNT_ID` must be set accordingly.
 - A convenience script `src/test/bash/primedb.sh` can be run to prime the DB with some exchange resources to use in manually testing:
+- Before exporting below environment variables manually set them in `src/test/bash/primedb.sh`
+
```
Comment (bmpotter) [inline README.md:70]: I don't understand this sentence. `primedb.sh` gets these variables from the environment at https://github.com/open-horizon/exchange-api/blob/b90546494646d231052e297d92120381d352e1b6/src/test/bash/primedb.sh#L40 . This way you don't need to change `primedb.sh`

File: README.md [hunk 3]
```diff
@@ -65,9 +66,12 @@ make gen-key
...
 ```
 export EXCHANGE_USER=<my-user-in-IBM-org>
 export EXCHANGE_PW=<my-pw-in-IBM-org>
+export  EXCHANGE_ROOTPW = <Exchange root password (Must be similar to what is set in exchange's config.json)>
```
Comment (bmpotter) [inline README.md:74]: The **Preconditions** section already tells them to set `EXCHANGE_ROOTPW`

**Review by bmpotter — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (megha070) [reply README.md:74]: Oh yes, I missed it.

File: README.md (hunk 1 above)
Comment (megha070) [reply README.md:56]: I need to confirm this once.

**Pull request closed without merging**